File: problemsets/Codeforces/C++/A747.cpp
/**
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 */
#include <bits/stdc++.h>
using namespace std;
int main() {
  int n; cin >> n;
  for (int i = sqrt(n+1); i >= 0; i--) if (i*(n/i) == n) {
    cout << i << " " << n/i;
    return 0;
  }
}

File: problemsets/Codeforces/Python/A747.py
"""
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 """
n=int(input())
x=1
while x*x<=n:
  if n%x<1:r=x
  x+=1
print(r,n//r)